src/components/pre-search/custom-query-preview.vue

```diff
@@ -3,6 +3,8 @@
     v-if="queriesPreviewInfo"
     :debounceTimeMs="250"
     :queriesPreviewInfo="queriesPreviewInfo"
+    :persistInCache="true"
+    :queryFeature="'customer'"
```

Comment:
The component has a prop for the query feature and the default value is already `'customer'`

```suggestion
    :queryFeature="queryFeature"
```